[CAL-2465] [CAL-2464] /api/trpc/insights/teamListForUser,userList,eventTypeList,userList,eventTypeList exceeds 4MB size

From axiom logs

```
PI response for /api/trpc/insights/teamListForUser,userList,eventTypeList,userList,eventTypeList?batch=1&input=%7B%220%22%3A%7B%22json%22%3Anull%2C%22meta%22%3A%7B%22values%22%3A%5B%22undefined%22%5D%7D%7D%2C%221%22%3A%7B%22json%22%3A%7B%22teamId%22%3A10542%2C%22isAll%22%3Afalse%7D%7D%2C%222%22%3A%7B%22json%22%3A%7B%22teamId%22%3A10542%2C%22userId%22%3A108666%2C%22isAll%22%3Afalse%7D%7D%2C%223%22%3A%7B%22json%22%3A%7B%22teamId%22%3A-1%2C%22isAll%22%3Afalse%7D%7D%2C%224%22%3A%7B%22json%22%3A%7B%22teamId%22%3Anull%2C%22userId%22%3A108666%2C%22isAll%22%3Afalse%7D%2C%22meta%22%3A%7B%22values%22%3A%7B%22teamId%22%3A%5B%22undefined%22%5D%7D%7D%7D%7D exceeds 4MB. API Routes are meant to respond quickly. https://nextjs.org/docs/messages/api-routes-response-size-limit
[ERROR] [1694661100348] LAMBDA_RUNTIME Failed to post handler success response. Http response code: 413.
```

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2464](https://linear.app/calcom/issue/CAL-2464/apitrpcinsightsteamlistforuseruserlisteventtypelistuserlisteventtypeli)</sub>

<sub>[CAL-2465](https://linear.app/calcom/issue/CAL-2465/[cal-2464])</sub>

--- a/packages/features/insights/components/TotalBookingUsersTable.tsx
+++ b/packages/features/insights/components/TotalBookingUsersTable.tsx
@@ -21,7 +21,7 @@ export const TotalBookingUsersTable = ({
                   <Avatar
                     alt={item.user.name || ""}
                     size="sm"
-                    imageSrc={item.user.avatar}
+                    imageSrc={"/" + item.user.username + "/avatar.png"}
                     title={item.user.name || ""}
                     className="m-2"
                   />

--- a/packages/features/insights/server/trpc-router.ts
+++ b/packages/features/insights/server/trpc-router.ts
@@ -73,7 +73,6 @@ const UserSelect = {
   id: true,
   name: true,
   email: true,
-  avatar: true,
   username: true,
 };
 

